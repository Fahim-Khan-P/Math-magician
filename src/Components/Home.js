import React from 'react';
import '../App.css';

const Home = () => (
  <div className="caculatorInfo">
    <h2>Welcome to our Simple Calculator Home page</h2>
    <p>
      A calculator is a device that performs arithmetic operations on numbers.
      Basic calculators can do only addition, subtraction, multiplication and
      division mathematical calculations.
      However, more sophisticated calculators can handle exponential operations,
      square roots, logarithms, trigonometric functions and hyperbolic functions. Internally,
      some calculators perform all these functions by repeated addition processes.
      This means that you can use these calculators to perform arithmetic operations on
      numbers.
      <br />
      <br />
      A calculator is a device that performs arithmetic operations on numbers.
      Basic calculators can do only addition, subtraction, multiplication and
      division mathematical calculations.
      However, more sophisticated calculators can handle exponential operations,
      square roots, logarithms, trigonometric functions and hyperbolic functions. Internally,
      some calculators perform all these functions by repeated addition processes.
      This means that you can use these calculators to perform arithmetic operations on
      numbers.
    </p>
  </div>
);

export default Home;
