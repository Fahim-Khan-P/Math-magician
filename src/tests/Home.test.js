import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Components/Home';

test('renders home page correctly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
