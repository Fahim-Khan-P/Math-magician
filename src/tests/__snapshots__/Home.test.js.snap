// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home page correctly 1`] = `
<div
  className="caculatorInfo"
>
  <h2>
    Welcome to our Simple Calculator Home page learn react
  </h2>
  <p>
    A calculator is a device that performs arithmetic operations on numbers. Basic calculators can do only addition, subtraction, multiplication and division mathematical calculations. However, more sophisticated calculators can handle exponential operations, square roots, logarithms, trigonometric functions and hyperbolic functions. Internally, some calculators perform all these functions by repeated addition processes.
    <br />
    OR
    <br />
    A calculator is a machine which allows people to do math operations more easily. For example, most calculators will add, subtract, multiply, and divide. There are simple basic calculators, scientific calculators & special purpose calculators (financial calculators etc.). Some also do square roots, and more complex calculators can help with calculus and draw function graphs.
  </p>
</div>
`;
